pkg/probe/fortiswitch_port_stats.go

```diff
@@ -0,0 +1,198 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func probeSwitchPortStats(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		mSwitchStatus = prometheus.NewDesc(
+			"fortiswitch_status",
+			"Whether the switch is connected or not",
+			[]string{"vdom", "name", "fgt_peer_intf_name", "connection_from", "state"}, nil,
+		)
+		mPortStatus = prometheus.NewDesc(
+			"fortiswitch_port_status",
+			"Whether the switch port is up or not",
+			[]string{"vdom", "name", "interface", "vlan", "duplex"}, nil,
```

Comment:
Can a port only have one VLAN on a
Fortiswitch? That seems odd. What happens when the port is a VLAN trunk?